cube/src/main/java/zone/cogni/asquare/cube/index/IndexMethod.java

```diff
@@ -211,11 +213,18 @@ private ObjectNode addFacets(Model draftModel, ObjectNode objectNode, String uri
     Map<String, RDFNode> bindings = ImmutableMap.of("uri", ResourceFactory.createResource(uri));
 
     ObjectNode facetNode = sparqlSelectToJson.convert(draftModel, bindings);
+    facetNode.put("hash", getHash(draftModel));
```

Comment:
this is pretty drastic. it would imply that all apps get this. whether they want it or not.

Reply:
ok, I checked. there is a `Configuration` class. we can use this to add a flag for adding hashes.